dags/main_summary.py

```diff
@@ -28,10 +30,19 @@
     env=tbv_envvar("com.mozilla.telemetry.views.MainSummaryView", {
         "from": "{{ ds_nodash }}",
         "to": "{{ ds_nodash }}",
+        "schema-report-location": main_summary_schema+"{{ ds_nodash }}",
         "bucket": "{{ task.__class__.private_output_bucket }}"}),
     uri="https://raw.githubusercontent.com/mozilla/telemetry-airflow/master/jobs/telemetry_batch_view.py",
     dag=dag)
 
+main_summary_schema = EmailSchemaChangeOperator(
+    task_id="main_summary_schema",
+    email=["telemetry-alerts@mozilla.com", "relud@mozilla.com"],
+    to=["telemetry-alerts@mozilla.com"],
```

Comment:
IIRC in the bug, we talked about limiting the `to` to folks that are specifically interested (you can add me! :D)